### PR TITLE
Minor documentation improvement in `measure.block_reduce`

### PR DESCRIPTION
**Pull request by justinkterry**
Small documentation change per discussion PR #4341.

### CODE REVIEW

File: skimage/measure/block.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def block_reduce(image, block_size, func=np.sum, cval=0, func_kwargs=None):
         block size.
     func_kwargs : dict
         Keyword arguments passed to `func`. Notably useful for passing dtype
-        argument to ``np.mean``.
+        argument to ``np.mean``. Takes dictionary of inputs, i.e.:
```
Comment (jni) [inline skimage/measure/block.py]:
```suggestion
        argument to ``np.mean``. Takes dictionary of inputs, e.g.:
```

**Comment by stefanv** [comment]
@justinkterry I would have added @jni's suggestion for you, but if you use your master branch to make PRs, then I cannot write to your branch.  No problem at all, but in the future remember to create a separate branch for the PR!  In the future, because we are always optimistic about great new contributors ;)

**Comment by justinkterry** [comment]
Merged JNI's suggestion, thanks a ton @stefanv

**Comment by emmanuelle** [comment]
Thank you @justinkterry , just waiting for the CI to roll and then I'll merge on green.

**Pull request merged**